### DISCUSSION TITLE
q-option-group: model should not be array: Why not?

### DISCUSSION THREAD

**Discussion by devslayertech** [quasarframework/quasar#14185] [General - Components / Directives / etc]
I used QOptionGroup in my project:
![image](https://user-images.githubusercontent.com/100967476/184413789-2403d6d5-d51e-4c49-a8df-8d1d33ddc3de.png)

It's works fine with this configuration but in console I have this error:
https://github.com/quasarframework/quasar/blob/97f2a09444b2a5f33bbd4e6693d3790c06bf80b2/ui/src/components/option-group/QOptionGroup.js#L60

Why not remove this error? QOptionGroup can work even using an array in the v-model.

**Comment by yusufkandemir** [comment]
You can use an array as the model for QOptionGroup **unless** it's the type of radio. The purpose of radio buttons is that only one is selectable, but with checkboxes, you can select as many as you want. When you use `type="radio"`, it uses QRadio, which uses `input[type="radio"]` under the hood. See [the HTML spec for `input[type="radio"]`](https://html.spec.whatwg.org/multipage/input.html#radio-button-state-(type=radio))
> The [input](https://html.spec.whatwg.org/multipage/input.html#the-input-element) element [represents](https://html.spec.whatwg.org/multipage/dom.html#represents) a control that, when used in conjunction with other [input](https://html.spec.whatwg.org/multipage/input.html#the-input-element) elements, forms a [radio button group](https://html.spec.whatwg.org/multipage/input.html#radio-button-group) **in which only one control can have its [checkedness](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#concept-fe-checked) state set to true**.
